.changeset/late-pots-smile.md

```diff
@@ -0,0 +1,9 @@
+---
+'@whatwg-node/server': minor
+---
+
+Fix context type to expose the `waitUntil` method.
+
+## Breaking Changes
+
+The function `handleNodeRequest` has been removed. Please use `handleNodeRequestAndResponse` instead.
```

Comment:
I am afraid this will be a breaking change on Yoga etc.

Reply:
Ah... That could be a problem... Do you think I should put it back ?

Reply:
Let's put it back until we cut a major release in Yoga.